server_tests/test/clientTests.js

```diff
@@ -1586,10 +1647,10 @@ describe("RUST API TEST SUITE", function() {
             return Q.all([testClient.bublStream(sessionId, function onStatusUpdate (res) {
                 if (!commandId) {
                     commandId = res.body.id;
-                    return testClient.bublStop(commandId).then( function onSuccess (res) {
+                    testClient.bublStop(commandId).then( function onSuccess (res) {
                         Comparison.bublStopOutput(res);
                         deferred.resolve();
-                    }).catch(deferred.reject);
+                    }).then(deferred.resolve, deferred.reject);
```

Comment:
The `resolve` one line above this isn't necessary, as it will be called twice in this case.